Dockerfile.graviton

```diff
@@ -0,0 +1,64 @@
+# Build the manager binary
```

Comment:
you might have to change this to something like this 

```
# syntax=docker/dockerfile:1.4
# Build the manager binary
FROM golang:1.23.0 AS builder

# Build-time arguments for platform and architecture
ARG TARGETOS
ARG TARGETARCH
ARG TARGETVARIANT

WORKDIR /workspace
# Copy the Go Modules manifests
COPY go.mod go.mod
COPY go.sum go.sum

# Cache deps before building and copying source so that we don't need to re-download as much
# and so that source changes don't invalidate our downloaded layer
RUN go mod download

# Copy the go source
COPY main.go main.go
COPY api/ api/
COPY controllers/ controllers/
COPY pkg/ pkg/
COPY tools/ tools/
COPY hack hack/

# Build the manager binary with dynamic OS/Arch support
RUN CGO_ENABLED=0 GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build -a -o manager main.go

# Multi-OS stage: Use a different base image depending on the ARG value
ARG BASE_IMAGE=ubi
ARG BASE_IMAGE_VERSION=8

# Use conditional `FROM` to select the base image
FROM ${BASE_IMAGE}:${BASE_IMAGE_VERSION} AS runtime

# Set environment variables
ENV OPERATOR=/manager \
    USER_UID=1001 \
    USER_NAME=nonroot

# Perform installation steps depending on the base image
RUN if [ "${BASE_IMAGE}" = "ubi" ]; then \
        yum -y install shadow-utils && \
        useradd -ms /bin/bash nonroot -u 1001 && \
        yum update -y krb5-libs && yum clean all && \
        yum -y update-minimal --security --sec-severity=Important --sec-severity=Critical && \
        yum -y update-minimal --security --sec-severity=Moderate && \
        yum -y update-minimal --security --sec-severity=Low; \
    elif [ "${BASE_IMAGE}" = "debian" ]; then \
        apt-get update && \
        apt-get install -y adduser && \
        adduser --uid 1001 --disabled-password --gecos "" nonroot && \
        apt-get clean; \
    elif [ "${BASE_IMAGE}" = "alpine" ]; then \
        apk add --no-cache shadow && \
        adduser -u 1001 -D nonroot; \
    fi

# Set labels for the image
LABEL name="splunk" \
      maintainer="support@splunk.com" \
      vendor="splunk" \
      version="2.2.1" \
      release="1" \
      summary="Simplify the Deployment & Management of Splunk Products on Kubernetes" \
      description="The Splunk Operator for Kubernetes (SOK) makes it easy for Splunk Administrators to deploy and operate Enterprise deployments in a Kubernetes infrastructure. Packaged as a container, it uses the operator pattern to manage Splunk-specific custom resources, following best practices to manage all the underlying Kubernetes objects for you."

WORKDIR /
RUN mkdir /licenses /tools/k8_probes

COPY --from=builder /workspace/manager .
COPY tools/EULA_Red_Hat_Universal_Base_Image_English_20190422.pdf /licenses
COPY LICENSE /licenses/LICENSE-2.0.txt
COPY tools/k8_probes/livenessProbe.sh /tools/k8_probes/
COPY tools/k8_probes/readinessProbe.sh /tools/k8_probes/
COPY tools/k8_probes/startupProbe.sh /tools/k8_probes/

USER 1001

ENTRYPOINT ["/manager"]
```

Reply:
Fixed